domain.yml

```diff
@@ -86,21 +86,19 @@ intents:
 
 slots:
   budget:
-    type: text
+    type: unfeaturized
   company_name:
-    type: text
+    type: unfeaturized
   email:
     type: text
```

Comment:
i **think** we should be able to use an unfeaturized entity here too now. But please double check